content/api/guides/appownerintegration/receivingdata/_index.md

```diff
@@ -36,11 +36,11 @@ Denne instansiering kan trigges av sluttbruker eller av tjenesteier. I denne gui
 
 Prosessen overordnet
 
-1. Sluttbruker instansierer tjeneste i parts meldingsboks i Altinn. Dette kan gjøres via API eller fra browser på Altinn.no
+1. Sluttbruker instansierer tjeneste i parts meldingsboks i Altinn. Dette kan gjøres via API eller i nettleser på Altinn.no
 2. Skjema fylles ut og eventuelle vedleggsdata lastes opp.
 3. Sluttbruker validerer data og eventuelle vedlegg og sender sender applikasjonsprosessen videre til bekreftelse
 4. Sluttbruker ser over data og bekrefter at han er ferdig med prosessen
-5. Applikasjon publiserer en hendelse om at sluttbruker er ferdig med utfyllingsprosess
+5. Applikasjon publiserer en hendelse om at sluttbruker er ferdig med utfyllingsprosess*
```

Comment:
Denne har vi jo sett at flere bommer på. Tenker det ikke kan gjentas for ofte at de må verifisere at tjenesten faktisk publiserer en hendelse. Så det sto nevnt lengere oppe også at det på enables for appen, men på vegne av alle som skumleser er ikke det noe man nødvendigvis får med seg.